core/client/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -161,6 +161,24 @@ public void hadoopShouldLoadFileSystemWhenConfiguredTest() throws Exception {
   }
 
   /**
+  * Ensures that when ListStatusNonExisting should Throw FileNotFoundException.
```

Comment:
`Ensures that FileNotFoundException is thrown when listStatus is called on a nonexistent path.`